pkg/monitors/grafana/grafana-monitor_test.go

```diff
@@ -0,0 +1,60 @@
+package grafana
+
+import (
+	"github.com/stakater/IngressMonitorController/v2/pkg/config"
+	"github.com/stakater/IngressMonitorController/v2/pkg/models"
+	"github.com/stakater/IngressMonitorController/v2/pkg/util"
+	"testing"
+)
+
+func TestAddMonitorWithCorrectValues(t *testing.T) {
+	config := config.GetControllerConfigTest()
+
+	service := GrafanaMonitorService{}
+	provider := util.GetProviderWithName(config, "Grafana")
+	if provider == nil {
+		return
+	}
+	service.Setup(*provider)
+	m := models.Monitor{Name: "google-test", URL: "https://google.com"}
+	service.Add(m)
+
+	mRes := service.GetAll()
+
+	if len(mRes) == 0 {
+		t.Errorf("Found empty response for Monitor. Name: %s and URL: %s", m.Name, m.URL)
+	}
+	if len(mRes) > 1 {
+		t.Errorf("Found too many response for Monitor, %v, after add.", len(mRes))
+	}
+	if mRes[0].Name != m.Name || mRes[0].URL != m.URL {
+		t.Error("URL and name should be the same", mRes[0], m)
+	}
+	m2 := models.Monitor{Name: "stakater-test", URL: "https://stakater.com", ID: mRes[0].ID, Config: mRes[0].Config}
+	service.Update(m2)
```

Comment:
can we explicitly separate out the checks for add update and delete like in other monitors?

Reply:
Sure thing, if that is more readable :eye:

Reply:
I spit the update and add test into 2, but since delete is needed for cleanup for both tests, it seemed weird to add another test to test something that is already testet